README.md

```diff
@@ -15,11 +15,28 @@ This extension currently only works on GitHub (and GitHub Enterprise), not on So
 1. Install the [Sourcegraph browser extension](https://docs.sourcegraph.com/integration/browser_extension).
    - In the Sourcegraph browser extension options menu, click the gear (⚙️) and enable **Experimental link previews** and **Experimental text field completion**.
 1. [Enable the Jira extension](https://sourcegraph.com/extensions/sourcegraph/jira) in the Sourcegraph extension registry (requires sign-in to Sourcegraph).
-1. Configure the Jira extension in [Sourcegraph user settings](https://sourcegraph.com/user/settings): `jira.apiToken`, `jira.url`, and `jira.username`.
+1. Configure the Jira extension in [Sourcegraph user settings](https://sourcegraph.com/user/settings): `jira.apiToken`, `jira.url`, and `jira.username`. See info below.
 1. Visit any text area, comment, PR, or issue on GitHub and try the [features](#features) below.
 
 If using GitHub Enterprise, right-click on the Sourcegraph browser extension icon when you're on a GitHub Enterprise page and select **Enable Sourcegraph on this domain**.
 
+## Configuring user settings
+
+The `jira.apiToken`, `jira.url`, and `jira.username` values should be added as top-level strings in the user settings (outside of the `extensions` object), eg:
```

Comment:
```suggestion
The `jira.apiToken`, `jira.url`, and `jira.username` values should be added as top-level keys in the user settings (outside of the `extensions` object), eg:
```

Reply:
@felixfbecker Great point for screen readers! Thank you. :)